File: leetcode/4.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        
        nums3 = nums1 + nums2 
        
        nums3.sort()
        
        if len(nums3) % 2 == 0:
            a = nums3[len(nums3) //2 ]
            b = nums3[len(nums3) // 2 - 1]
            print((a+b) / 2) 
            return (a + b) / 2
        
        else:
            a = len(nums3) // 2
            
            print(a)
            return nums3[a]

File: leetcode/4.js
var findMedianSortedArrays = function (nums1, nums2) {
  const mergedArray = [...nums1, ...nums2].sort((a, b) => a - b)
  let median

  // if length of merged array is even then get the mid two numbers and find median
  if (mergedArray.length % 2 === 0) {
    median =
      (mergedArray[mergedArray.length / 2 - 1] +
        mergedArray[mergedArray.length / 2]) /
      2
  } else {
    // if length is odd then mid number is median
    median = mergedArray[Math.floor(mergedArray.length / 2)]
  }

  return median
}
